hermes-tracker-elasticsearch/src/test/java/pl/allegro/tech/hermes/tracker/elasticsearch/consumers/ConsumersElasticsearchLogRepositoryTest.java

```diff
@@ -45,13 +47,13 @@ public class ConsumersElasticsearchLogRepositoryTest extends AbstractLogReposito
     private static ElasticsearchResource elasticsearch = new ElasticsearchResource();
     private SchemaManager schemaManager;
 
-    @BeforeSuite
+    @BeforeClass
```

Comment:
The newer version of testng does not support injection for @BeforeSuite https://testng.org/#_dependency_injection